modules/order/src/OrderListBuilder.php

```diff
@@ -110,4 +110,22 @@ public function buildRow(EntityInterface $entity) {
     return $row + parent::buildRow($entity);
   }
 
+  /**
+   * {@inheritdoc}
+   */
+  protected function getDefaultOperations(EntityInterface $entity) {
+    $operations = parent::getDefaultOperations($entity);
+
+    if ($entity->access('update') && $entity->hasLinkTemplate('reassign-form')) {
```

Comment:
I left the "hasLinkTemplate" in case someone overrides entity class, or removes this action in favor of something else. Then they do not have to override this as well (if they did not want to.)